fiftyone/core/odm/database.py

```diff
@@ -788,7 +788,8 @@ def bulk_write(ops, coll, ordered=False):
         ordered (False): whether the operations must be performed in order
     """
     try:
-        for ops_batch in fou.iter_batches(ops, 100000):  # mongodb limit
+        batch_size = fo.config.bulk_write_batch_size
```

Comment:
The mock in your PR description made me realize that users may appreciate the ability to see progress bars for certain invocations of this.

Sometimes there are multiple bulk writes in a single public SDK method, so it likely wouldn't make sense to show progress bars for each of them because it becomes unclear when the method will actually finish (or at least I've avoided doing this for the most part to-date).

However, `set_values()` is a great example of a case where there's just one call to `bulk_write()`.

We could add an optional `progress=True` flag to this method which `set_values()` and other choice methods could utilize.

Reply:
(for future work)